test/classes/ferguson/class-double-modifier.good

```diff
@@ -34,6 +34,3 @@ class-double-modifier.chpl:24: error: duplicate decorators - shared shared A(int
 class-double-modifier.chpl:23: In function 'doit3':
 class-double-modifier.chpl:24: error: duplicate decorators - shared borrowed A(int(64))
   class-double-modifier.chpl:58: called as doit3(type C = borrowed A(int(64)))
-class-double-modifier.chpl:30: In function 'doit4':
```

Comment:
I'm surprised more wasn't removed from this .good file, given the lines you removed.

Reply:
Ah you are right, this test does need more changes, I guess I missed this one in my local testing